src/todo.js

```diff
@@ -1,29 +1,20 @@
-import React from 'react';
-
-const ToDo= (value,
```

Comment:
tumi akol ata parameter puwa eyate, but tumi 3 parameter accept koribo karone likhi thoisa. Since okol 1st parameter't sob value ahi asil. heiye error throw kori asil